com.unity.render-pipelines.universal/Runtime/2D/Passes/Render2DLightingPass.cs

```diff
@@ -89,7 +89,7 @@ private void CopyCameraSortingLayerRenderTexture(ScriptableRenderContext context
                 ClearFlag.None, Color.clear);
             cmd.SetGlobalTexture(k_CameraSortingLayerTextureID, m_Renderer2DData.cameraSortingLayerRenderTarget.nameID);
             context.ExecuteCommandBuffer(cmd);
-            CommandBufferPool.Release(cmd);
```

Comment:
can ExecuteCommandBuffer be removed from here since it happens also later? Or it's important that Execute and Clear happen here?

Reply:
will investigate this

Reply:
let's leave this to not affect the cmdbuffer syncing. these are hard to test issues as they could work on one platform but not the other and are hard to track and test

Reply:
I would think we don't need the execute & clear here or elsewhere most of the time. 

If I remember correctly, execute just takes a copy of the buf in the context and clear resets the local one.
Instead just collect everything into the buf and submit once. 

But it's completely ok to do this in another PR. 
There might have been some ownership issues with the cmdbufs.
Anyway, we just should add task for removing and testing these properly.

Reply:
there are several issues with removing these calls, the main culprit being ProfilingScopes and clashes with Context calls. I guess we could do a lot of these calls cleanup when we change context.DrawRenderers and similar calls with cmd.DrawRendererLists. I suggest now to play it safe and keep these calls, and cleanup in a single sweep when we can